lib/src/repositories/connect_bluetooth_device_repository.dart

```diff
@@ -2,33 +2,15 @@ part of '../../viam_flutter_bluetooth_provisioning_widget.dart';
 
 class ConnectBluetoothDeviceRepository {
   BluetoothDevice? get device => _device;
-  Stream<BluetoothConnectionState?> get bluetoothConnectionStateStream => _stateController.stream;
-  BluetoothConnectionState? get bluetoothConnectionState => _bluetoothConnectionState;
-  set bluetoothConnectionState(BluetoothConnectionState? state) {
-    if (_bluetoothConnectionState != state) {
-      _bluetoothConnectionState = state;
-      _stateController.add(state);
-    }
-  }
-
-  final StreamController<BluetoothConnectionState?> _stateController = StreamController<BluetoothConnectionState>.broadcast();
-  BluetoothConnectionState? _bluetoothConnectionState;
-  StreamSubscription<BluetoothConnectionState>? _connectionStateSubscription;
```

Comment:
All of this was unused, I had added it for some future use case but it just made me confused really.